packages/env-provider/src/services/index.ts

```diff
@@ -0,0 +1,107 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { IEnvProviderService } from './IEnvProviderService';
+import dotenv from 'dotenv';
+import findConfig from 'find-config';
+
+export class EnvProviderService implements IEnvProviderService {
```

Comment:
Let's make this shorter: `EnvProviderService` -> `EnvProvider`

Reply:
I know am late to the comments, Could it even be `ConfigService`? does it make more sense?

Reply:
Renamed.